.github/workflows/build.yml

```diff
@@ -29,4 +30,8 @@ jobs:
       if: matrix.rust == 'stable'
 
     - name: tests
-      run: cargo test --all 
+      run: cargo test --all
+
+    - name: display free disk space
```

Comment:
Do we really need this now?

Reply:
No, but I think it would be a good idea to keep on eye on free disk space for a while.